articles/reporting/metrics-definitions.md

```diff
@@ -36,9 +36,9 @@ The percentage of messages that are bounced by SparkPost - suppression logic i
 
 Number of messages that are bounced by SparkPost - suppression logic is the most common type of admin bounce.
 
-**All Bounce Rate**
+**Bounce Rate**
 
-The percentage of messages that bounce (for any reason).  Calculated by *number of bounces / number of messages targeted*
+The percentage of messages that bounce except admin bounces.  Calculated by *(number of bounces - admin bounces ) / number of messages sent*
```

Comment:
is "number of bounces - admin bounces" correct?  we changed the way we count bounces so if I look at the webui or make an api call and do Bounces - Admin Bounces I think that's the wrong number

Reply:
That should be correct, I could update this to say `number of in band bounces + number of out of band bounces / number of messages sent`.  That would be correct since we exclude admin bounces from in band now.